packages/xs-vat-worker/package.json

```diff
@@ -30,7 +30,6 @@
     "ava": "^3.12.1",
     "detective-es6": "^2.2.0",
     "eslint": "^6.8.0",
-    "eslint-config-airbnb": "^18.0.1",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-config-jessie": "^0.0.4",
     "eslint-config-prettier": "^6.9.0",
```

Comment:
We might be able to elide some more of these eslint* dependencies since they’re brought in by `@agoric/eslint-config` and not used directly.